### ISSUE TITLE
召集uu参与教程编写

### ISSUE THREAD

**Issue by NoFish-528** [datawhalechina/thorough-pytorch#27]
为了更好的完善教程，我们结合当前thorough-pytorch项目的进度，给出了以下task，希望有更多的同学能加入开源的贡献。主要有以下任务：

补充深度学习相关知识（包括卷积，池化，全连接，激活函数，dropout，batch normalization，BP反向传播）（2人）
补充计算机视觉领域、自然语言处理领域，语音处理相关简介（类比一篇论文的introduction，只需说明是什么，有哪些方法即可）（多人）
介绍人工智能发展简史、DL,ML,AI三者之间的关系（1人）
介绍评价相关指标（混淆矩阵， accuracy， precision， recall， F1 score）（1人）
Jupyter notebook或Jupyter Lab相关操作（1人）
一些经典代码的解读（包括但不限于ResNet，GoogleNet，MobileNet，RCNN系列，SSD，YOLO系列，transformer系列，ViT，Detr，BERT，GPT系列，CLIP，GAN相关）（该任务意在介绍一些真正的代码工作是如何写的，建议解读的是PyTorch源码、官方实现的代码或者一些标准库代码包括但不限于timm等）（多人）
联系方式：
email: [nzk020109@163.com](mailto:nzk020109@163.com)
wechat: ZhikangNiu_Wechat

**Comment by andongBlue** [comment]
可以基于transformer模型做N，对NLP领域的Pytorch实战，做进一步补充
